src/app/layout/qgslayoutimagedrophandler.cpp

```diff
@@ -0,0 +1,73 @@
+/***************************************************************************
+    qgslayoutimagedrophandler.cpp
+    ------------------------------
+    begin                : November 2019
+    copyright            : (C) 2019 by nyall Dawson
+    email                : nyall dot dawson at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgslayoutimagedrophandler.h"
+#include "qgslayoutdesignerinterface.h"
+#include "qgslayout.h"
+#include "qgsreadwritecontext.h"
+#include "qgsproject.h"
+#include "qgslayoutview.h"
+#include "qgslayoutitempicture.h"
+#include <QMessageBox>
```

Comment:
That include (as well as the #include "qgsreadwritecontext.h" and #include "qgsproject.h" above) are not needed, should be removed.